.dockerignore

```diff
@@ -0,0 +1,8 @@
+.env
+.venv
+.env.template
+.env.test
+Vagrantfile
+logs.txt
+.git
+.vagrant
```

Comment:
This is a pretty good list but there are still a few more things you could include like Dockerfile, and README.md.
But that's somewhat irrelevant given that you are being very specific with your COPY commands in the Dockerfile anyway.

The only things that are currently being copied that you could ignore are the generated __pycache__ folders.

Reply:
Done, thanks.